src/main/java/fivium/pat/utils/PatAuthUtils.java

```diff
@@ -0,0 +1,282 @@
+package fivium.pat.utils;
+
+import static fivium.pat.utils.Constants.CLINICIAN_PORTAL_LOGIN_CONTEXT;
+import static fivium.pat.utils.Constants.MOBILE_APP_LOGIN_CONTEXT;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.mindrot.jbcrypt.BCrypt;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+
+public class PatAuthUtils {
+
+	private static final String JWT_CONTEXT_CLAIM_KEY = "login_context";
```

Comment:
Consider moving these constants to the constants file.

Reply:
The Constants file should only contain constants that have a scope of more than one class